docker/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM ubuntu:20.10
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Get the basic stuff
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y \
+    sudo
+
+# Create ubuntu user with sudo privileges
+RUN useradd -ms /bin/bash ubuntu && \
+    usermod -aG sudo ubuntu
+# New added for disable sudo password
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# Set as default user
+USER ubuntu
+WORKDIR /home/ubuntu
+
+# Install opam
+RUN sudo apt install -y rsync git m4 build-essential patch unzip wget pkg-config libgmp-dev libev-dev libhidapi-dev libffi-dev opam jq zlib1g-dev curl
+RUN curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh > /tmp/install.sh
+RUN ["/bin/bash", "-c", "sudo /bin/bash /tmp/install.sh <<< /usr/local/bin"]
+RUN opam init -y --disable-sandboxing --bare
+RUN echo "test -r /home/ubuntu/.opam/opam-init/init.sh && . /home/ubuntu/.opam/opam-init/init.sh > /dev/null 2> /dev/null || true" >> /home/ubuntu/.profile
+
+# Install Rust
+RUN wget https://sh.rustup.rs/rustup-init.sh
+RUN chmod +x rustup-init.sh
+RUN ./rustup-init.sh --profile minimal --default-toolchain 1.44.0 -y
+ENV PATH="$HOME/.cargo/bin:${PATH}"
```

```suggestion
ENV PATH="/home/ubuntu/.cargo/bin:${PATH}"
```

Untested, but something like this might help if you cant find rustc in the path

---

Yeah. That's the fix @Engil also suggested. :-) That seems to work.